src/worker/jobs/chainSubscribers.ts

```diff
@@ -415,9 +529,17 @@ class ChainSubscribers {
 
     async _processOtherEvents(log: ethers.providers.Log): Promise<void> {
         try {
-            const parsedLog = this.ifaceCommunity.parseLog(log);
+            let parsedLog: ethers.utils.LogDescription;
+            if (log.address === config.communityContractAddress) {
+                parsedLog = this.ifaceNewCommunity.parseLog(log);
+            } else {
+                parsedLog = this.ifaceCommunity.parseLog(log);
+            }
             if (parsedLog.name === 'ManagerAdded') {
-                const managerAddress = parsedLog.args[0];
+                const managerAddress =
+                    parsedLog.args.length > 1
+                        ? parsedLog.args[1]
+                        : parsedLog.args[0];
```

Comment:
It's better to revert this. With the new contracts we can achieve the same without this.

Reply:
I didn't understand, the new contract send 2 parameters, and the manager's address come in position 1. The old contract just send one parameter, with the manager's address in position 0